src/external-repos.md

```diff
@@ -114,7 +117,7 @@ the week leading up to the beta cut.
 
 Rust's build system builds a number of tools that make use of the internals of
 the compiler and that are hosted in a separate repository, and included in Rust
-via git submodules (such as [Miri]). If these tools break because of your
+via git submodules (such as [Cargo]). If these tools break because of your
```

Comment:
I think Cargo does not make use of internals of compiler, so the first sentence of this paragraph may need changing.

Reply:
I ended up removing this section altogether because we already document this on forge and link to it in the previous section: https://forge.rust-lang.org/infra/toolstate.html